geode-core/src/integrationTest/java/org/apache/geode/cache/client/internal/AutoConnectionSourceImplJUnitTest.java

```diff
@@ -313,7 +296,7 @@ public void test_DiscoverLocators_whenOneLocatorWasShutdown() throws Exception {
       ArrayList<ServerLocation> locators = new ArrayList<>();
       locators.add(new ServerLocation(InetAddress.getLocalHost().getHostName(), secondPort));
       handler.nextLocatorListResponse = new LocatorListResponse(locators, false);
-      Thread.sleep(500);
+      Thread.sleep(1500);
```

Comment:
I think we should try to replace this sleep with a use of await. What background activity is happening during the call to sleep?

Reply:
Good call. I figured out what to await for here to allow the test to pass.